src/components/Accordion/Accordion.stories.tsx

```diff
@@ -26,6 +25,7 @@ import { EXAMPLE_IMAGES } from '../AssetInput/example-assets';
 import { Accordion as AccordionComponent, AccordionItem } from './Accordion';
 import { AccordionHeaderIcon } from './AccordionHeaderIcon';
 import { AccordionHeaderIconSize, AccordionHeaderProps, AccordionProps } from './types';
+import IconTextAlignmentCentre from '@foundation/Icon/Generated/IconTextAlignmentCentre';
```

Comment:
IconTextAlignmentCentre should that not be IconTextAlignmentCenter, or was there the need to have separate names?